src/components/benefitsHub/ConnectWithUsPanel.tsx

```diff
@@ -0,0 +1,100 @@
+import { BenefitsHub as FormattedBenefitsHub } from './formatted-type'
+
+type ConnectWithUsProps = {
+  connectWithUs: FormattedBenefitsHub['connectWithUs']
+}
+export function ConnectWithUsPanel({ connectWithUs }: ConnectWithUsProps) {
+  const socialLinksObject =
+    connectWithUs.field_social_media_links.platform_values
+  return (
+    <va-accordion-item open bordered header="Connect with us" level="2">
+      <h3 className="vads-u-font-size--h4">Get updates</h3>
```

Comment:
I'm not sure what the story is, but in prod, it looks like there's a link before this first heading. Can you confirm that it's not supposed to be there?

<img width="324" height="155" alt="Image" src="https://github.com/user-attachments/assets/5c4e69f0-7d41-42cb-b400-3342b4f222bc" />